api_spec.yaml

```diff
@@ -19,11 +19,11 @@ paths:
     post:
       description: Returns model response based on input parameters
       requestBody:
-        description: File with input parameters
+        description: JSON with input parameters
         content:
-          text/plain:
+          application/json:
             schema:
-              type: string
+              type: object
```

Comment:
Add details (object fields). Better allow a list of object, batch mode.

Reply:
done